index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
Please, you have one script tag at the begging of your body tag and your other script tag, but at the end of your body tag, I recommend you put them together because it would be nice to maintenance your code cleaner, more professional, and easier to understand in a professional environment. In the checkbox below, you can see an article about where is the best way for putting your script tags in HTML.

- [ ]  Put together your scripts tags (At the end of your HEAD tag or BODY tag) You can check [this article](https://www.homeandlearn.co.uk/javascript/javascript_tag_placement.html) to see why deeper.